disnake/webhook/async_.py

```diff
@@ -1076,6 +1079,11 @@ async def foo():
         Only given if :attr:`type` is :attr:`WebhookType.channel_follower`.
 
         .. versionadded:: 2.0
+
+    application_id: Optional[:class:`int`]
+        The ID of the application that created this webhook.
```

Comment:
`...if it was created by an application.`

Reply:
fixed in 75ae9e1e84ac6e3098cc85651940fba2953e8449